pgcli/packages/function_metadata.py

```diff
@@ -1,7 +1,7 @@
 import re
 import sqlparse
 from sqlparse.tokens import Whitespace, Comment, Keyword, Name, Punctuation
-
+from ..pgcompleter import ColumnMetadata
```

Comment:
We have a rule of thumb that stuff in the /packages module should be independent of pgcli. So it'd be better to move the definition of ColumnMetadata here, and have pgcompleter import it, instead of the other way around